### PR TITLE
19 leo's additions

### PR DESCRIPTION
**Pull request by MangoMagoCane**

### CODE REVIEW

File: frontend/src/components/Learn Lessons/learnObj.tsx
Comment (jvgusch) [inline frontend/src/components/Learn Lessons/learnObj.tsx:1]: Filename should be capitalized. And maybe named differently? "LearnObj" is pretty vague, without looking at the file I'm unable to guess at what it contains.

**Review by JudeKlatte — DISMISSED** [review]

File: frontend/src/components/LearnButtonSelector.tsx
Comment (jvgusch) [inline frontend/src/components/LearnButtonSelector.tsx:1]: Maybe this name needs rethought as well? The name implies that it just deals with buttons but it also displays the lesson text when an lesson is open.

**Review by jvgusch — APPROVED** [review]
My comments are super nit-picky, but overall the change looks good! While I recommend considering renaming a couple of the files I'm not going to hold up the review for that, especially with the demo coming up this afternoon.

File: frontend/src/components/Learn Lessons/learnObj.tsx
Comment (MangoMagoCane) [reply frontend/src/components/Learn Lessons/learnObj.tsx:1]: There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors.

Comment (MangoMagoCane) [reply frontend/src/components/Learn Lessons/learnObj.tsx:1]: Updated to `learnAppDirectory`

**Review by josephmaxwellhyland — APPROVED** [review]
Looks good to me!

**Pull request merged**